### PR TITLE
document inspecting webviews

### PR DESCRIPTION
**Pull request by lieut-data**
As per a [comment](https://community.mattermost.com/core/pl/q69g9hs7rjd3idrmqtqnqgo1to) from @svelle linking out to https://github.com/electron/electron/issues/2285, it turns out we can open the developer tools for individual webviews fairly easily. 

This PR captures the details and some screenshots to illustrate. Further productization here might include a way to launch these developer tools from the desktop app interface for easier discoverability, as well as documentation for how developer tool plugins might be installed (e.g. React devtools).

### CODE REVIEW

File: site/content/contribute/desktop/debugging.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+date: 2019-01-22T00:00:00-05:00
+weight: 3
+subsection: "Debugging"
+---
+
+# Developer Tools
+
+The electron app itself can be inspected using the developer tools, available from the View menu:
+
+![image](/contribute/desktop/view-toggle-developer-tools.png)
```
Comment (cometkim) [inline site/content/contribute/desktop/debugging.md]: I suggest using available description for here for site accessibility.

```suggestion
![View toggle developer tools](/contribute/desktop/view-toggle-developer-tools.png)
```

File: site/content/contribute/desktop/debugging.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+However, this view does not expose the contents of the webviews for inspection. To open the developer tools for a webview, first access the console for the electron shell as above, and then run
+
+    document.getElementById("mattermostView0").openDevTools();
```
Comment (cometkim) [inline site/content/contribute/desktop/debugging.md]: Not important, but just for consistency with other codeblocks?

```suggestion
```js
document.getElementById("mattermostView0").openDevTools();
```.
```

File: site/content/contribute/desktop/debugging.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+using `mattermostView0` for the first opened tab, `mattermostView1` for the second opened tab, etc. A new window will appear with the developer tools for that webview:
+
+![image](/contribute/desktop/webview-developer-tools.png)
```
Comment (cometkim) [inline site/content/contribute/desktop/debugging.md]: Also here

```suggestion
![Webview developer tools](/contribute/desktop/webview-developer-tools.png)
```

**Comment by svelle** [comment]
> Would there be any benefit in changing the desktop app to do this automatically, or would it be best to leave it as it is?

IMO there would be, but there still needs to be the option to open the dev console for the desktop app itself too (obviously). So maybe a button somewhere in the view-menu that would open the webview console would be good? Could be placed next to the regular dev console button.

**Comment by lieut-data** [comment]
Sounds like an awesome help-wanted ticket idea! I'll write one up to complement (and eventually replace) this PR.

**Comment by lieut-data** [comment]
https://mattermost.atlassian.net/browse/MM-13827 for the help wanted ticket.

**Pull request merged**